Sources/carton/Commands/Dev.swift

```diff
@@ -32,9 +32,6 @@ struct Dev: ParsableCommand {
   @Option(help: "Specify name of an executable product in development.")
   var product: String?
 
-  @Option(help: "Specify name of a json destination file to be passed to `swift build`.")
```

Comment:
Just a suggestion here, let's keep the flag and notify users that this option no longer has any effect and will be removed in a future version. This will give some transition period to whoever was still using it. WDYT?